Jenkinsfile_CNP

```diff
@@ -53,11 +53,7 @@ withPipeline(type, product, component) {
   env.S2S_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
 
   before('functionalTest:preview') {
-    env.URL_TO_TEST = "http://am-lib-test-aat.service.core-compute-aat.internal"
-  }
-
-  before('functionalTest:aat') {
-    env.URL_TO_TEST = env.TEST_URL
+    env.environment_name ="preview"
```

Comment:
Is this required? The stage name already has "preview" as the environment

Reply:
Yes accessing it while inserting data.